### ISSUE TITLE
commentFilePath missing from DSL

### ISSUE THREAD

**Issue by sjthespian** [jenkinsci/ghprb-plugin#514]
It appears that when the comment file option was added to the plugin, it was never added to the DSL. I have yet to figure out how to add it via. groovy, so is it possible to get this added to the DSL? This option is also missing from the global settings for the plugin, so we can't even add a default for all jobs on the server.

In the meantime, does anyone have any suggestions for adding this via. groovy? I've been trying various variations on the following with no success:
```groovy
          configure { node ->
              def extensions = node / 'triggers' / 'org.jenkinsci.plugins.ghprb.GhprbTrigger'('plugin': 'ghprb@1.36.2') / 'extensions'
              extensions << 'org.jenkinsci.plugins.ghprb.extensions.comments.GhprbCommentFile' {
                  commentFilePath('github-comments.txt')
              }
          }
```

**Comment by ramc-n** [comment]
I have been looking for this as well.
Could some one please help look into this?
We are able to add commentFilePath when we create the job manually, but using DSL we are unable to add commentFilePath.
Thank You.

**Comment by bjoernhaeuser** [comment]
Closed by #637
